configure.ac

```diff
@@ -54,10 +49,9 @@ dnl Running tests with Valgrind is slow. It is faster to iterate on
 dnl code without Valgrind until tests pass, then enable Valgrind and
 dnl fix leaks.
 AC_ARG_ENABLE([valgrind],
-   AS_HELP_STRING([--disable-valgrind],[do not run tests under Valgrind]))
+   AS_HELP_STRING([--enable-valgrind],[enable Valgrind during testing]))
```

Comment:
Just to understand: With this configure will not check if valgrind is actually available when using `--enable-valgrind`?

Reply:
I see, the check in tests/setup? as i understand the will silently skip using valgrind if missing?

Reply:
Exactly. The behavior is not changed in this PR.

Reply:
Ok, i guess it could possible fail if valgrind was explicitly enabled but missing? but if so could be done in a separate PR i think